helm-chart/metabase/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "metabase.fullname" . }}
+  labels:
+    {{- include "metabase.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.externalPort }}
+      targetPort: http
```

Comment:
Hi can I check if this is supposed to be
{{ .Values.service.internalPort }} - in order words port 3000?

Reply:
it's now fixed in the latest commit